commercetools-models/src/main/java/io/sphere/sdk/products/ProductDraftBuilder.java

```diff
@@ -158,66 +71,13 @@ public ProductDraftBuilder categories(final List<Reference<Category>> categories
      */
     public ProductDraftBuilder categoriesAsObjectList(final List<Category> categories) {
         final List<Reference<Category>> referenceList = categories.stream()
-                .filter(x -> x != null)
-                .map(cat -> cat.toReference())
+                .filter(Objects::nonNull)
+                .map(Category::toReference)
                 .collect(Collectors.toList());
         return categories(referenceList);
     }
 
-    public ProductDraftBuilder searchKeywords(final SearchKeywords searchKeywords) {
-        this.searchKeywords = searchKeywords;
-        return getThis();
-    }
 
-    public ProductDraftBuilder categoryOrderHints(@Nullable final CategoryOrderHints categoryOrderHints) {
-        this.categoryOrderHints = categoryOrderHints;
-        return getThis();
-    }
-
-    public ProductDraftBuilder key(@Nullable final String key) {
-        this.key = key;
-        return getThis();
-    }
-
-    public LocalizedString getName() {
-        return name;
-    }
-
-    public LocalizedString getSlug() {
-        return slug;
-    }
-
-    @Nullable
-    public LocalizedString getDescription() {
-        return description;
-    }
-
-    @Nullable
-    public LocalizedString getMetaTitle() {
-        return metaTitle;
-    }
-
-    @Nullable
-    public LocalizedString getMetaDescription() {
-        return metaDescription;
-    }
-
-    @Nullable
-    public LocalizedString getMetaKeywords() {
-        return metaKeywords;
-    }
-
-    public Set<Reference<Category>> getCategories() {
-        return categories;
-    }
-
-    public SearchKeywords getSearchKeywords() {
-        return searchKeywords;
-    }
-
-    public Reference<TaxCategory> getTaxCategory() {
-        return taxCategory;
-    }
 
     public ProductDraftBuilder taxCategory(final Referenceable<TaxCategory> taxCategory) {
```

Comment:
This method and the `state`are already generated, so we should remove them here.